sbncode/CAFMaker/FillTrigger.cxx

```diff
@@ -4,23 +4,29 @@
 namespace caf
 {
   void FillTrigger(const sbn::ExtraTriggerInfo& addltrig_info,
-		   const std::vector<raw::Trigger>& trig_info,
-		   std::vector<caf::SRTrigger>& triggerInfo)
+		   const raw::Trigger& trig,
+		   caf::SRTrigger& triggerInfo)
   {
     uint64_t beam_ts = 0;
     uint64_t trigger_ts = 0;
-    triggerInfo.emplace_back();
-    triggerInfo.back().global_trigger_time = addltrig_info.triggerTimestamp;
-    triggerInfo.back().beam_gate_time_abs = addltrig_info.beamGateTimestamp;
+    triggerInfo.global_trigger_time = addltrig_info.triggerTimestamp;
+    triggerInfo.beam_gate_time_abs = addltrig_info.beamGateTimestamp;
     beam_ts = addltrig_info.beamGateTimestamp;
     trigger_ts = addltrig_info.triggerTimestamp;
```

Comment:
I am surprised that the compiler does not complain about these two variables being "set but not used" or something like that. I suggest they are just removed (including their definitions above at lines 10-11).
```suggestion
```

Reply:
Fixed.